vault/logical_system.go

```diff
@@ -347,7 +371,7 @@ func NewSystemBackend(core *Core, config *logical.BackendConfig) (logical.Backen
 			},
 
 			&framework.Path{
-				Pattern: "revoke-force/(?P<prefix>.+)",
+				Pattern: "(lease/)?revoke-force" + framework.OptionalParamRegex("prefix"),
```

Comment:
You are right.  I got a little ambitious in my refactoring.  I'll revert out these changes.